packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -125,10 +125,7 @@ type AccessibilityRoles = {
  * <code>import "@ui5/webcomponents-fiori/dist/FlexibleColumnLayout.js";</code>
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.fiori.FlexibleColumnLayout
- * @extends sap.ui.webc.base.UI5Element
- * @tagname ui5-flexible-column-layout
+ * @extends UI5Element
  * @public
  * @since 1.0.0-rc.8
```

Comment:
- remove namespace from layout-change event
- remove `@type` tag
- remove `@sap-restricted` tag from _layoutsConfiguration
- remove `@readonly` and `@type` tags from getters and add explicit return type